python/ray/actor.py

```diff
@@ -1685,6 +1685,16 @@ def _remote(self, args=None, kwargs=None, **actor_options) -> ActorProxy[T]:
             function_signature = meta.method_meta.signatures["__init__"]
             creation_args = signature.flatten_args(function_signature, args, kwargs)
 
+        use_placement_group = scheduling_strategy is not None and isinstance(
+            scheduling_strategy, PlacementGroupSchedulingStrategy
+        )
+        if use_placement_group and detached:
+            logger.warning(
+                "Scheduling a detached actor with a placement group is not recommended, "
```

Comment:
As @robertnishihara pointed out, detached placement groups are a thing.  You did highlight a scenario for placement group deletion, but I think it'd be more pertinent to give a warning on that scenario separately (deleting a detached placement group).

The above may give a user the impression that you just shouldn't use placement groups with actors, but that is absolutely something we support.